src/app/shared/datatypes/asset.interface.ts

```diff
@@ -81,12 +81,14 @@ export class Asset {
             if (data.fieldName === 'License' && data.fieldValue === 'ADL Terms and Conditions') {
                 data.fieldValue = 'Use of this image is in accordance with the <a href="https://www.artstor.org/artstor-terms/" target="_blank">Artstor Terms & Conditions</a>'
             }
-            // Split by semicolons without removing semicolon
-            let values = data.fieldValue.split('|')
+            
+            let values = Array.isArray(data.fieldValue) ? data.fieldValue : [data.fieldValue]
```

Comment:
Can we no longer assume that `data.fieldValue` will be a string?

Reply:
We should be able to, I had been experimenting with passing other values

Reply:
Cleaned it out 👍